src/menu.c

```diff
@@ -3963,7 +3963,7 @@ void render_file_select_menu(s32 arg0) {
     func_8009BD5C();
     func_80067F2C(&D_801263A0, &D_801263A8);
     for (i = 0; i < 3; i++) { // 3 files
-        if (D_801264A0[i].unk0 == gIsInAdventureTwo || D_801264A0[i].unk1 == 0) {
```

Comment:
This needs to have the `unk0` and `unk1` fixed to their respective new names from the struct to fix NON_MATCHING builds.

Reply:
Done. Thanks!